bentoml/yatai/client/bento_repository_api.py

```diff
@@ -51,13 +57,74 @@
     safe_retrieve,
     load_from_dir,
 )
+from bentoml.yatai.proto.yatai_service_pb2_grpc import YataiStub
 from bentoml.yatai.status import Status
 
 
 logger = logging.getLogger(__name__)
 yatai_proto = LazyLoader('yatai_proto', globals(), 'bentoml.yatai.proto')
 
 
+class BentoUploadStreamRequests:
+    """
+    A class for iterating over a file to generate upload bento requests
+
+    Args:
```

Comment:
I think conventionally this part of the docstring goes under the `__init__` method?